src/validation/overseas.entity.query.validation.ts

```diff
@@ -0,0 +1,12 @@
+import { body } from "express-validator";
+
+import { ErrorMessages } from "./error.messages";
+import { VALID_OE_FORMAT } from "./regex/regex.validation";
+
+export const overseasEntityQuery = [
+  body("oe_number")
+    .not().isEmpty({ ignore_whitespace: true })
+    .withMessage(ErrorMessages.OE_QUERY_NUMBER)
+    .matches(VALID_OE_FORMAT)
+    .withMessage(ErrorMessages.INVALID_OE_NUMBER_LENGTH),
```

Comment:
The message should be invalid OE number! It's no longer a check for length only.